src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,177 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node node;
+    private Node head = null;
+    private Node tail = null;
+    private int size = 0;
+
     @Override
     public void add(T value) {
+        node = new Node<>(null, value, null);
```

Comment:
The constructor call for `Node` should include the previous node (tail in this case) and the next node should be `null`, which you can omit as it's the default value.